### PR TITLE
Reflecting refactor of umbral.umbral to umbral.pre, some touched-up docstrings and code org.

### PR DESCRIPTION
**Pull request by jMyles**

### CODE REVIEW

File: nkms/network/protocols.py [hunk 1]
```diff
@@ -55,6 +55,7 @@ def determine_legality_of_dht_key(self, signature, sender_pubkey_sig,
         verified = signature.verify(hrac, sender_pubkey_sig)
 
         if not verified or not proper_key == dht_key:
+            # Hachidan Kiritsu, it's illegal!
```
Comment (tuxxy) [inline nkms/network/protocols.py]: .. I... hmm... ?

File: nkms/network/server.py [hunk 2]
```diff
@@ -62,14 +62,14 @@ def serialize_capabilities(self):
 
     def get_now(self, key):
```
Comment (tuxxy) [inline nkms/network/server.py]: Do you think we could rename this to something more self documenting?

Comment (jMyles) [reply nkms/network/server.py]: Hmm - sure!  I just took `get` and made it happen synchronous / blocking (ie, "get it *now*").  

What do you suggest instead?

File: nkms/network/protocols.py (hunk 1 above)
Comment (KPrasch) [reply nkms/network/protocols.py]: Anbo-jyutsu

File: nkms/network/server.py (hunk 2 above)
Comment (tuxxy) [reply nkms/network/server.py]: Ah, I guess that makes sense then. It just feels weird to simply see `self.get` or `self.get_now`.

I think I prefer `get_<thing>_now` so it's more clear, but this is fine too.

Comment (jMyles) [reply nkms/network/server.py]: Yeah... I'm just not sure what to make <thing>.  :-)

`get_value_now`?  I'm not sure if `value` adds anything to me or not, but I guess it does reduce the initial cryptic phrasing.

**Pull request merged**